test/ReportPortal.Shared.Tests/Helpers/MockServiceBuilder.cs

```diff
@@ -13,16 +13,16 @@ public Mock<IClientService> Build()
         {
             var service = new Mock<IClientService>();
 
-            service.Setup(s => s.Launch.StartAsync(It.IsAny<StartLaunchRequest>())).Returns(() => new ValueTask<LaunchCreatedResponse>(new LaunchCreatedResponse { Uuid = Guid.NewGuid().ToString() }));
```

Comment:
I think, that here for testing porpose we can keep returning instance of ValueTask. What do you think? Becuase we know, that operation will be performed synchronously.

Reply:
Signature is returning Task.